crates/transaction-emitter-lib/src/emitter/mod.rs

```diff
@@ -419,8 +420,8 @@ impl TxnEmitter {
                     req.max_account_working_set,
                     req.gas_price,
                 )),
-                TransactionType::NftMint => Box::new(
-                    NFTMintGeneratorCreator::new(
+                TransactionType::NftMintAndTransfer => Box::new(
+                    NFTMintAndTransferGeneratorCreator::new(
                         self.from_rng(),
                         txn_factory.clone(),
                         root_account,
```

Comment:
but you are passing root_account here ? 

I don't see anywhere that we are creating a separate creator_account, or am I missing something?